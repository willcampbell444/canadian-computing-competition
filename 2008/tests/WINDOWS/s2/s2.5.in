19999
20000
25000
0
