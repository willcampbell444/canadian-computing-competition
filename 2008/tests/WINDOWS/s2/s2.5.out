1256510833
1256636857
1963494901
